drivers/bluetooth/hci/hci_nxp.c

```diff
@@ -369,6 +472,13 @@ int bt_nxp_setup(const struct device *dev, const struct bt_hci_setup_params *par
 			break;
 		}
 #endif /* CONFIG_HCI_NXP_ENABLE_AUTO_SLEEP */
+#if CONFIG_BT_HCI_SET_PUBLIC_ADDR
```

Comment:
use if( IS_ENABLED(CONFIG_BT_HCI_SET_PUBLIC_ADDR)) to increase compilation coverage

Reply:
I made the change to use `if( IS_ENABLED()` instead of `#if` where it's possible. Thanks